18/README.md

```diff
@@ -0,0 +1,208 @@
+```
+shortname: [18/TEP]
+name: Transactional Election Process
+type: standard
+status: raw
+editor: Alberto Granzotto <alberto@bigchaindb.com>
+```
+
+
+# Transactional Election Process
+
+## Abstract
+<!-- The abstract is a short (~200 word) description of the technical issue being addressed. -->
+This specification introduces the new concept of **Election**. An Election is an asynchronous process that when successful triggers Network wide changes synchronously (i.e. at the same *block height*). An Election is started by any Validator in the Network, called **Initiatior**. The Election itself and all cast Votes are transactions, hence stored in the blockchain. This enables new Validators to replay Network changes incrementally, while syncing.
+
+## Motivation
+<!--The motivation is critical for BEPs that want to change the BigchainDB protocol. It should clearly explain why the existing protocol BEP is inadequate to address the problem that the BEP solves. BEP submissions without sufficient motivation may be rejected outright.-->
+Changing the shape of a BigchainDB Network at runtime is an important requirement. While this BEP does not address this issue directly, it wants to solve the limitations we had with [Upsert Validators][BEP-3] by giving a tool that can be used this and other situations.
```

Comment:
> it wants to solve the limitations we had with [Upsert Validators][BEP-3]

I would add a more detailed explanation of the limitations.

As I see them:

- Since Validator Set changes are not themselves expressed as transactions but are included into the metadata of an arbitrary block, all the nodes have to add a validator simultaneously - if less than the majority of the nodes attempt to include a validator, the block fails. Not only the attempt to add a validator fails but all the transactions in the block have to go through the consensus process again. The proposed algorithm, on the other hand, consists of the individual transactions each node can verify without making itself a decision to include a validator.
- Nodes fail to replay the blocks committed by the validators they do not know yet. The `Invalid commit (last validator set size) vs (number of precommits)` problem.